src/main/java/com/xpanxion/java/springboot/da1/demo/model/student5/WorkoutHistory5.java

```diff
@@ -0,0 +1,84 @@
+package com.xpanxion.java.springboot.da1.demo.model.student5;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name="workout_history5")
+public class WorkoutHistory5 {
+
+    @Id
+    @SequenceGenerator(
+            name = "workout_id_sequence",
+            sequenceName = "workout_id_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "workout_id_sequence"
+    )
+    private long workoutId;
+    private long memberId;
```

Comment:
Do you think we should bind this to a Member object instead of a long?

Reply:
Fixed this and also bound a Gym object to the Member.